README.md

```diff
@@ -4,6 +4,9 @@ This package enables Node.js applications to use a CAN bus over USB.
 
 ## Making a release
 
-1. Update `version` field in `package.json`
-2. Push to GitHub
-3. Run `npm run publish`
+1. Check out the `main` branch
+2. Update `version` field in `package.json`
```

Comment:
I would add that they should also run `npm install`, so that the version numbers between `package.json` and `package-lock.json` match

Reply:
Good call